resources/assets/coffee/lang-js.coffee

```diff
@@ -0,0 +1,29 @@
+###
+#    Copyright 2015-2017 ppy Pty. Ltd.
+#
+#    This file is part of osu!web. osu!web is distributed with the hope of
+#    attracting more community contributions to the core ecosystem of osu!.
+#
+#    osu!web is free software: you can redistribute it and/or modify
+#    it under the terms of the Affero GNU General Public License version 3
+#    as published by the Free Software Foundation.
+#
+#    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+#    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#    See the GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+
+# Lang.js monkey patch
+Lang.origGet = Lang.get
+Lang.get = (key, replacements, locale) ->
```

Comment:
Looking again the arguments can probably be removed and call the original method with `.apply null, arguments` instead.

The workaround in `osu.trans` can also be removed.